compiler/src/prog.ml

```diff
@@ -123,7 +123,7 @@ type ('len,'info,'asm) gfunc = {
     f_body : ('len,'info,'asm) gstmt;
     f_tyout : 'len gty list;
     f_outannot : Annotations.annotations list; (* annotation attach to return type *)
-    f_ret  : 'len gvar_i list
+    f_ret  : 'len gvar_i list L.located
```

Comment:
I have the impression that the change will be much smaller if you add the location in the field f_outannot.
Something like f_outannot : L.t * Annotations.annotations list;
This will allow to not have to patch all occurence of f_ret.
Then maybe f_outannot is not a good name anymore and can renamed.

Reply:
Thanks for this good advice. I’ve just implemented it.